crates/ra_vfs/src/io/mod.rs

```diff
@@ -0,0 +1,233 @@
+use std::{
```

Comment:
We use 2018 style, so `io.rs` instead of `io/mod.rs`

Reply:
I didn't know that, but I like it more.